modules/programs/alot-accounts.nix

```diff
@@ -1,4 +1,4 @@
-{ config, lib, ... }:
+{ config, lib, pkgs, ... }:
```

Comment:
To do the `pkgs` passing I believe you need something like
```nix
pkgs:
{ config, lib, ... }:
```
here instead. Then to change
```nix
(import ../programs/alot-accounts.nix)
```
in `email.nix` to
```nix
(import ../programs/alot-accounts.nix pkgs)
```

Reply:
seems to work though it feels a bit ugly thanks !